l1-contracts/contracts/bridgehub/Bridgehub.sol

```diff
@@ -465,12 +474,19 @@ contract Bridgehub is IBridgehub, ReentrancyGuard, Ownable2StepUpgradeable, Paus
 
     /*//////////////////////////////////////////////////////////////
                         Chain migration
-    //////////////////////////////////////////////////////////////*/
+    //////////////////////////////////////////////////////////////
+
+    Methods below are used when we're moving a 'child' chain between diffent layers
+    For example from L1 to Gateway.
+    In these cases, we treat the child chain as an 'asset' - where we burn it in the 'source bridgehub'
```

Comment:
Then we should add a comment that technically `assetId` is the asset ID of the STM, not chain per se